### PR TITLE
[hi]Localize content/en/docs/reference/glossary/application-developer.md

### PR DESCRIPTION
**Pull request by taniaduggal**
Signed-off-by: Tania Duggal <103496926+taniaduggal@users.noreply.github.com>

Fixes #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/application-developer.md
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/application-developer.md:7]:
```suggestion
 एक व्यक्ति जो Kubernetes क्लस्टर में चलने वाले एप्लिकेशन लिखता है।
```

File: content/hi/docs/reference/glossary/application-developer.md
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/application-developer.md:13]:
```suggestion
 एक व्यक्ति जो Kubernetes क्लस्टर में चलने वाले एप्लिकेशन लिखता है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Slight grammatical nit.

**Comment by taniaduggal** [comment]
@divya-mohan0209 I made suggested changes. I request you please fast review and merge it, as I am willing to participate in 1.26 release shadow program ,so at least some pr should be merge is a plus point in my application.
thanks:)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by divya-mohan0209 — COMMENTED** [review]
Looks good to me!

/lgtm

**Comment by taniaduggal** [comment]
/assign @divya-mohan0209 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by divya-mohan0209** [comment]
@taniaduggal I understand that you need this as a supplement to your application and I appreciate the check-in. However, please bear in mind that we're working through a backlog of docs reviews here and it's not just the Hindi localization that I'm talking about when I say that. As the docs co-chair, I'd request you to be empathetic and patient if your PR hasn't received a review in a few hours because there are other things that are a priority like the release artifacts/blogs that need to be published over the next week/two. We're all trying our best here to get to each PR submitted while fostering a healthy environment for all contributors. Would request that you wait for atleast a couple of days before pinging for a re-review either here or on Slack. Please do let me know if any questions.

File: content/hi/docs/reference/glossary/application-developer.md
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/application-developer.md:7]:
```suggestion
 एक व्यक्ति जो कुबेरनेट्स क्लस्टर में चलने वाले एप्लिकेशन लिखता है।
```

File: content/hi/docs/reference/glossary/application-developer.md
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/application-developer.md:13]:
```suggestion
 एक व्यक्ति जो कुबेरनेट्स क्लस्टर में चलने वाले एप्लिकेशन लिखता है।
```

File: content/hi/docs/reference/glossary/application-developer.md
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/application-developer.md:17]:
```suggestion
एप्लिकेशन डेवलपर्स एप्लिकेशन के एक हिस्से पर ध्यान केंद्रित करते हैं। उनकी कार्य सीमा के आकार में एक उल्लेखनीय अंतर हो सकता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/application-developer.md
Comment (anubha-v-ardhan) [reply content/hi/docs/reference/glossary/application-developer.md:7]: We can localize Kubernetes.

**Comment by anubha-v-ardhan** [comment]
> I have apply all changes @anubha-v-ardhan :)

Thank you, if you could squash the commits, we can get this pr ready for merge.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by divya-mohan0209** [comment]
Hello @taniaduggal - please squash ALL the commits & let us know. We'll get another set of eyes to review & approve.

**Comment by divya-mohan0209** [comment]
Closing this PR since it is a duplicate of #38332 
/close

**Pull request closed without merging**
